modal.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <script src="https://kit.fontawesome.com/0a707a45d9.js" crossorigin="anonymous"></script>
+    <title>Modal</title>
+</head>
+<body>
+    <div class="modal-container">
+        <section class="modal-text-container">
```

Comment:
the parent container should be given the `section` tag and `div` should be given to its children

Reply:
okay...will look into that....thank you for feedback